docs/accessibility.md

```diff
@@ -0,0 +1,89 @@
+
+## Automated Testing/Checking
+
+Our ESLint rules include some basic accessibility checks using the [jsx-a11y plugin](https://github.com/evcohen/eslint-plugin-jsx-a11y).
+
+## Resources
+
+A collection of resources that will introduce you to the concept of accessibility and some of the standards behind it.
+
+* [WebAIM](http://webaim.org/): WebAIM has provided comprehensive web accessibility solutions since 1999, and is one of the leading providers of web accessibility expertise internationally. WebAIM is a non-profit organization within the [Center for Persons with Disabilities](http://www.cpd.usu.edu/) at [Utah State University](http://www.usu.edu/). The site brings together a huge collection of information about web accessibility. Most of it is up to date and evergreen enough to help you. The [organization's blog](http://webaim.org/blog/) tackles many modern-day, evolving topics related to web accessibility.
+* [W3C’s Web Accessibility Initiative](http://www.w3.org/WAI/): The World Wide Web Consortium's Web Accessibility Initiative provides strategies, guidelines and resources to make the Web accessible to people with disabilities.
+* [A11y Buzz](http://www.a11ybuzz.com/): Several web folks in the accessibility community launched this site in January 2012. It's a collection of accessibility resources that are triaged and reviewed by experts before they're posted.
+* [The Six Simplest Accessibility Tests Anyone Can Do](http://www.karlgroves.com/2013/09/05/the-6-simplest-web-accessibility-tests-anyone-can-do/): A handful of simple things you can do to think about and implement accessibility with little effort as you work on projects.
+* [Why Keyboard Accessibility Isn’t the Same Thing as Screen Reader Accessibility](https://www.ssbbartgroup.com/blog/why-keyboard-accessibility-isnt-the-same-thing-as-screen-reader-accessibility/): This blog post explains the three critical aspects of accessibility for interactive web technologies: Keyboard Accessibility, Screen Reader Accessibility and Cognitive Accessibility, and the differences between each.
+* [Teaching Accessibility Core Rules](http://www.deque.com/pragmatica11y-teaching-accessibility-core-rules): This post breaks down accessibility principles to their core, making them even easier to think about during projects.
+* Presentation: [Explorations in the Virtual DOM: How React.js Impacts Accessibility](https://marcysutton.github.io/react-a11y-presentation/)
+* [Inclusive Design Fundamentals](https://isner.github.io/inclusive-design-fundamentals/), especially the [disability types](https://isner.github.io/inclusive-design-fundamentals/handouts/disability-types.html) to learn about how different disabilities affect how people use technology.
+
```

Comment:
Let's add the [Inclusive Components project](https://inclusive-components.design/) to this list too, they have a growing list of component patterns for things like menus, tooltips, etc along with information on why to use what ARIA properties.